src/EnergyPlus/HVACUnitarySystem.cc

```diff
@@ -6945,155 +6935,567 @@ namespace HVACUnitarySystem {
 			// if no load, or only a moisture load which can't be met at PLR=1, RETURN
 			return;
 		}
-		// must test to see if load is bounded by capacity before calling RegulaFalsi
-		if ( ( HeatingLoad && ZoneLoad < SensOutputOn ) || ( CoolingLoad && ZoneLoad > SensOutputOn ) ) {
```

Comment:
These next pink code were not deleted, they are now part of he ELSE after the IF ASHRAE model code...see below at ESLE